doc/nrf/samples/samples_zigbee.rst

```diff
@@ -3,6 +3,78 @@
 Zigbee samples
 ##############
 
+The nRF Connect SDK provides the following samples showcasing the :ref:`ug_zigbee` protocol.
+The samples can be built for various boards and can be configured for different usage scenarios.
+
+In addition to their basic functionalities, the samples can be expanded with variants and extensions.
+The following table lists variants and extensions available for each Zigbee sample:
+
+.. list-table::
+    :widths: auto
+    :header-rows: 1
+
+    * - Variant or extension
+      - Light bulb
+      - Light switch
+      - Network coordinator
+      - NCP
+      - Shell
+      - Template
+    * - FEM support
+      - ✔
+      - ✔
+      - ✔
+      - ✔
+      - ✔
+      -
+    * - Sleepy End Device behavior
+      -
+      - ✔
+      -
+      -
+      - :ref:`** <zigbee_ug_sed>`
+      - :ref:`** <zigbee_ug_sed>`
+    * - Multiprotocol Bluetooth LE
+      - :ref:`*** <ug_multiprotocol_support>`
+      - ✔
+      - :ref:`*** <ug_multiprotocol_support>`
+      -
```

Comment:
Technically, `BLE` can be enabled for `NCP` sample, we have tested combination of Zigbee NCP + BLE HCI dongle some time ago.

Reply:
OK, is the configuration procedure the same as for other samples?

Reply:
No, it is not. It deserves at least 4x`*` in my opinion:
 - BLE application cannot be simply added since the NCP SoC has no direct access to the application logic.
 - A proper BLE application may be implemented, but must utilize custom commands through NCP transport.
 - The second option is to add the BLE serialization through the HCI USB endpoint, but this is far from what's described in the linked user guide.
 - It is unclear how someone can integrate BLE application through HCI with the existing NCP Host application. It is a valid configuration and it is (theoretically) supported, but we do not provide a sample that shows how to do this.

Generally: If we want to add it, let's put at least 4 stars and a link to the BLE HCI guides instead of the multiprotocol one.

Reply:
@tomchy Let's assume that if someone is savvy enough, they would know how to make multiprotocol `Zigbee NCP` sample so they don't need `****` in this table and so do we.